components/typography/styles.js

```diff
@@ -52,7 +52,7 @@ export const bodyStandardStyles = css`
  * A private helper method that should not be used by general consumers
  */
 export const _generateBodyCompactStyles = (selector) => {
-	if (!_isValidCssSelector(selector)) return;
+	// if (!_isValidCssSelector(selector)) return;
```

Comment:
Do you recall why we explicitly didn't allow this the complex selector case?

https://github.com/BrightspaceUI/core/blob/main/components/typography/test/styles.test.js#L28

Reply:
Oh it was [this](https://github.com/BrightspaceUI/core/pull/2698#discussion_r931527209). Maybe be we can allow complex selectors, but not if they include `:host`.

Reply:
There was no explicit reason we didn't include more complex cases. I think we wanted to start with a minimum viable solution, and not build more than we needed at the time.

Just curious, what's the danger of allowing complex selectors that include `:host`?